api/graphql/mutation/calendarEventRequest/index.js

```diff
@@ -21,7 +21,7 @@ class CalendarEventRequest extends BaseMutationResolver {
       },
     });
 
-    return form.create();
+    return event?.is_open && form.create();
```

Comment:
No need to check for is_open here. 

Move the validation logic from [CalendarEventInviteForm](https://github.com/coding-blocks/gondor/blob/master/api/services/CalendarEventInvite/Form.js#L15) to [AuthorizationPolicy](https://github.com/coding-blocks/gondor/blob/master/api/services/AuthorizationPolicy/index.js#L123). And also add the is_open check in that function.

Study this [lib](https://github.com/hereisnaman/auth-policy) to understand how auth policy works.

Reply:
Can you please explain a little more what i am supposed to do here?